<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgSecLocators</class>
 <widget class="QWidget" name="wgSecLocators">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SecLocators</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/shelf_AR.jpg</normaloff>img/shelf_AR.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);
border-bottom-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTwistJointsCount">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Twist Joints Count</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editTwistJointsCount">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreateForearmTwist">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Create Forearm Twist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreateReverseFootroll">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Create Reverse Footroll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteSecLocators">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>Delete SecLocators</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
